hirosystems/stacks-blockchain/templates/statefulset.yaml

```diff
@@ -180,6 +180,9 @@ spec:
             - name: tcp-metrics
               containerPort: {{ .Values.containerPorts.metrics }}
               {{- end }}
+            { { - if .Values.extraPorts } }
+            { { - include "common.tplvalues.render" (dict "value" .Values.extraPorts "context" $) | nindent 12 } }
+            { { - end } }
```

Comment:
Small nit, would you mind formatting the braces like they are elsewhere, without the empty space inbetween?
`{{` vs `{ {`

Reply:
cleaned up